app/Jobs/Mship/SyncToForums.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace App\Jobs\Mship;
+
+use App\Models\Mship\Account;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\DB;
+
+class SyncToForums implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    private $account;
+
+    public function __construct(Account $account)
+    {
+        $this->account = $account;
+    }
+
+    public function handle()
```

Comment:
Should we consider putting a concern on the `Account` model that the account `HasForumAccount`?